server/command.go

```diff
@@ -852,6 +861,17 @@ func executeInstanceInstallServer(p *Plugin, c *plugin.Context, header *model.Co
 	})
 }
 
+func checkIfJiraIsAccessible(jiraURL string) (string, bool) {
+	jiraIsAccessible, err := utils.IsJiraAccessible(jiraURL)
+	if err != nil {
+		return err.Error(), false
+	}
+	if !jiraIsAccessible {
+		return "It looks like we couldn't validate the connection to your Jira server. Please make sure the URL was entered correctly. This could also be because of existing firewall or proxy rules. If you intend to have a one way integration from Jira to Mattermost this is not an issue.", false
```

Comment:
Can we have this constant at the top of the file?

Reply:
Done.